docusaurus/docs/recipes/rest-api.md

```diff
@@ -4,18 +4,32 @@ title: Adding a simple REST API
 sidebar_label: Adding a simple REST API
 ---
 
-Sometimes a project may require a simple REST API (e.g. to send an email). Next.js comes with the ability to create API endpoints, which makes this recipe very straightforward. Please read their [API routes](https://nextjs.org/docs#api-routes) documentation as they extend Node.js `req` and `res` objects with additional functionality and ship with built-in middleware.
+Sometimes a project may require a simple REST API (e.g. to send an email). Next.js brought API routes support in v9, but you have to provide your own implementation for handling different HTTP methods, validation, error handling and so on. We have created [`@moxy/next-rest-api`](https://github.com/moxystudio/next-rest-api/) to solve these problems, so that you can focus on writing your business logic.
+
+> ℹ️ Besides getting familiar with the `next-rest-api`, you might want to give a quick read over Next.js [API routes](https://nextjs.org/docs#api-routes) documentation. Next.js extends Node.js `req` and `res` objects with additional functionality and ships with built-in middleware.
 
 > ⚠️ Please note that if you require more than a simple API with one or two endpoints, it's better if you create a separate project (and repository) for it.
 
 ## Walk-through
 
+### 1. Install `@moxy/next-rest-api`
+
+```bash
+npm i @moxy/next-rest-api @hapi/joi @hapi/boom
+```
+
 ### 1. Create `api/my-endpoint.js` file
```

Comment:
`2.`

We should also update de others.

Reply:
Fixed.

Reply:
Also, it might be me, but should we be more clear that we must create an `api` folder at the root of our project?

I think this brief explanation could be useful even for the `pages` mapping explanation.